src/compiler/crystal/semantic/to_s.cr

```diff
@@ -79,5 +79,10 @@ module Crystal
       @str << node.value
       false
     end
+
+    def visit(node : Unreachable)
```

Comment:
Shouldn't the to_s here remove the `else unreachable` entirerly to preserve better the semantics of the input code?

Reply:
Yes, maybe. The thing is that to_s is never called in the compiler for unreachable, just for debug purposes.

Reply:
What I mean is, the Unreachable node is generated when expanding case into if else. There's no else we can remove from the case because it's gone. If we remove the else from the if then we get something nilable. raise is the most similar translation.

Reply:
Ok, i see. My worries is that some code instrumentation might loose exhaustive case check properties. But probably i am over worrying here.